src/CountUp.tsx

```diff
@@ -119,49 +125,65 @@ class CountUp extends Component {
     // end value.
     if (end !== prevProps.end) {
       if (!preserveValue) {
-        this.instance.reset();
+        this.instance!.reset();
       }
-      this.instance.update(end);
+      this.instance!.update(end);
     }
   }
 
   componentWillUnmount() {
     if (this.timeoutId) {
       clearTimeout(this.timeoutId);
     }
-    this.instance.reset();
+    if (this.instance) {
+      this.instance.reset();
+    }
   }
 
   createInstance = () => {
     if (typeof this.props.children === 'function') {
       // Warn when user didn't use containerRef at all
       warning(
-        this.containerRef.current &&
-          (this.containerRef.current instanceof HTMLElement ||
-            this.containerRef.current instanceof SVGTextElement ||
-            this.containerRef.current instanceof SVGTSpanElement),
+        !!(
+          this.containerRef.current &&
+          AVAILABLE_TAGS.includes(
+            this.containerRef.current.tagName.toLowerCase(),
+          )
+        ),
         `Couldn't find attached element to hook the CountUp instance into! Try to attach "containerRef" from the render prop to a an HTMLElement, eg. <span ref={containerRef} />.`,
       );
+      return null;
     }
-    return createCountUpInstance(this.containerRef.current, this.props);
+    return createCountUpInstance(
+      this.containerRef.current as HTMLElement,
+      this.props,
+    );
   };
 
   pauseResume = () => {
     const { reset, restart: start, update } = this;
     const { onPauseResume } = this.props;
 
-    this.instance.pauseResume();
+    if (this.instance) {
+      this.instance.pauseResume();
+    }
```

Comment:
```suggestion
      this.instance?.pauseResume();
```

I'm unsure if we included optional chaining in babel.